WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -177,6 +188,8 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
                 postEditorUtil.handlePublishButtonTap()
             case .close:
                 postEditorUtil.cancelEditing()
+            case .more:
+                displayMoreSheet()
```

Comment:
It feels weird to "request html" to show the action sheet.

I understand that it's needed for the secondary publish button and to update the preview, but maybe we could reuse the `.publish` reason and create a new one for `preview`?

We might then need new `requestHTMLReason` for every `more` action that requires the latest HTML. 
🤔 

What do you think?

Reply:
We actually need it to decide if we should show "Save as Draft" option in the action sheet. That option is shown only if there's any change in editor. And we might also need it for the History option also. So basically we need the last state of the html in order to construct options we need to show in action sheet. This is the basic idea behind it, so what do you think?

Reply:
Right! That makes complete sense. ✅